src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowSpec.groovy

```diff
@@ -511,9 +511,19 @@ class VxlanFlowSpec extends HealthCheckSpecification {
     def "Unable to create a vxlan flow when dst switch does not support it"() {
         given: "VXLAN supported and not supported switches"
         def switchPair = topologyHelper.getAllNeighboringSwitchPairs().find {
-            switchHelper.isVxlanEnabled(it.src.dpId) && !switchHelper.isVxlanEnabled(it.dst.dpId)
+            switchHelper.isVxlanEnabled(it.src.dpId) &&
+                    (!switchHelper.isVxlanEnabled(it.dst.dpId) || switchHelper.isVxlanEnabled(it.dst.dpId))
```

Comment:
This is an always-true expression, makes no sense

Reply:
fixed